appserver/pom.xml

```diff
@@ -153,7 +153,7 @@
 
         <!-- MicroProfile Config -->
         <microprofile.config-api.version>3.0.2</microprofile.config-api.version>
-        <helidon-config.version>3.0.2</helidon-config.version>
+        <helidon-config.version>3.1.0</helidon-config.version>
```

Comment:
helidon upgrades that may not be necessary is mixed in.

Reply:
From my perspective it would be alright just to rename the PR including descriptions.

Reply:
We could do both, but I'll revert the Helidon change then. Thx!